### ISSUE TITLE
Error using sits_cube function using local cube

### ISSUE THREAD

**Issue by joaowerner** [e-sensing/sits#606]
Hello, I'm trying to use the functionality of the SITS package (Version: 0.16.2-1), but I can't open my images stored on my disk. In the R documentation about the sits_cube function there is no current information on how to open data cubes from a directory using a new function or another. For now, the error that appears to me is related to the function and not to the images. The purpose is for academic purposes of my PhD research, working with 10-day composition cubes (Sentinel-2 and PlanetScope).

Console error message: "Error in sits_cube.local_cube(source = "LOCAL", name = "DataFusion", data_dir = data_dir, :
   LOCAL value is deprecated. Please see the documentation on ?sits_cube"

![image](https://user-images.githubusercontent.com/73180269/154137847-5aa8127d-e020-428a-90b0-a6bef445a8d8.png)

Best regards;

**Comment by rolfsimoes** [comment]
Hi @joaowerner ,

The `sits_cube(source = "LOCAL", ... )` call is deprecated. 
In the current version, you must pass to the `source` and `collection` parameters one of the sources/collections supported by sits:
```R
sits_list_collections()
```
sits uses these two parameters to load metadata from its internal database, such as `no_data`, `maximum_value` etc.
A list of metadata used by sits can be read using the following command:
```R
sits_config_show(source = "BDC", collection = "CB4_64-1")
```
In this example you can see all the metadata for the source `"BDC"` and the collection `"CB4_64-1"`.
You need to find out which metadata corresponds to your local files.

To access the local files, you must pass a valid path to the `data_dir` parameter (as you have done). In this path, sits will look for files and retrieve tile, band and date information from the filenames.
To do this, sits uses the `delim` and `parse_info` parameters.

If you are still having difficulties opening a local cube, please let me know the satellite/sensor of your images and an example of the name of an image file in the folder given in `data_dir`. Also, the output of the `sessionInfo()` function will be helpful.

Best regards,
Rolf

**Comment by joaowerner** [comment]
Hi @rolfsimoes 

Thanks for your clarifications. I managed to open my local cube providing within collections the correspondence "S2-SEN2COR_10_16D_STK-1". An example of the name of my images in the directory (data_dir) is "ST_C10_NDVI_2018-09-01" for Sentinel-2 and "PS_C10_NDVI_2018-09-01" for Planet. In this case, I set the delim = "_" and parse_info = c("X1","tile","band", "date"). I was also able to apply the sits_get_data() function normally. The database for each of the different satellites/sensors followed the definitions of a data cube described in the paper "Satellite Image Time Series Analysis for Big Earth Observation Data". 

I would like to do analysis for classification within the SITS package, exploring all features. So it came to my mind that the difference in spatial resolution between my Planet composite product (3 meters) and the one offered in "S2-SEN2COR_10_16D_STK-1" (10 meters) could cause future problems, is this possible?

The output of the sessionInfo():
![image](https://user-images.githubusercontent.com/73180269/154551845-b73b2607-978b-4ffd-a5d3-a07b3788ab3e.png)

Best regards;
João Paulo
